checks/evaluation/security_policy.go

```diff
@@ -15,10 +15,59 @@
 package evaluation
 
 import (
+	"fmt"
+
 	"github.com/ossf/scorecard/v4/checker"
 	sce "github.com/ossf/scorecard/v4/errors"
 )
 
+func scoreSecurityCriteria(contentLen, linkedContentLen, urls, emails, discvuls int) (int, string) {
+	score := 0
+	reason := ""
+
+	// #1: found one linked (email/http) content: score += 3
+	//     rationale: someone to collaborate with or link to
+	//     information (strong for community)
```

Comment:
Maybe add a link to an example .md file that satisfies this, to help readers of the code.

Reply:
So, in this latest commit there are two unit-test cases which I used as exemplars `10_realworld` (earns 10 points) and `07_realworld` (earns 7 points) pulled from `kubernetes` and `ossf/scorecard` respectively.

When you say "add a link" - do you mean to the `checks.yaml` which would get incorporated into `checks.md` (by way of `make generate-docs`? Or something else?

Reply:
I meant an http link to an actual repository that satisfies it. Maybe that's not needed.